components/server/src/bitbucket-server/bitbucket-server-repository-provider.ts

```diff
@@ -210,7 +210,7 @@ export class BitbucketServerRepositoryProvider implements RepositoryProvider {
 
     public async searchRepos(user: User, searchString: string): Promise<RepositoryInfo[]> {
         // Only load 1 page of 10 results for our searchString
-        const results = await this.api.getRepos(user, { maxPages: 1, limit: 10, searchString });
+        const results = await this.api.getRepos(user, { maxPages: 1, limit: 30, searchString });
```

Comment:
bumped this to 30 as 10 were a bit too few. We still should somehow communicate to users that there is more or alternatively fetch more when the user has scrolled down.